pgcli/main.py

```diff
@@ -103,16 +103,13 @@ def cli(database, user, password, host, port):
             try:
                 _logger.debug('sql: %r', document.text)
                 res = pgexecute.run(document.text)
-                output = []
```

Comment:
I haven't had a chance to review your diff yet, but you're right it is a bug. 

The click.echo_via_pager() call should be outside the for loop. Good catch.